python_program/main.py

```diff
@@ -17,19 +18,36 @@ def generate_automated_walk_table(departure_date, gpx_file_path, velocity):
 
     # calc Points for walk table
     total_distance, temp_points, way_points = select_waypoints(raw_gpx_data)
-    plot_elevation_profile(raw_gpx_data, way_points, temp_points, file_name=name)
+    plot_elevation_profile(raw_gpx_data, way_points, temp_points, file_name=name, open_figure=open_figure)
     create_walk_table(departure_date, velocity, way_points, total_distance, file_name=name)
-    plot_route_on_map(raw_gpx_data, way_points, file_name=name)
+    plot_route_on_map(raw_gpx_data, way_points, file_name=name, open_figure=open_figure, map_scaling=map_scaling)
 
 
 if __name__ == "__main__":
-    # defines the departure time of the hike
-    DEPARTURE_TIME = datetime(year=2021, month=8, day=16, hour=9, minute=00)
-
-    # Path to the GPX-File with the pre-planed route
-    GPX_FILE_PATH = './GPX/CHANGE_ME.gpx'
-
-    # Velocity , walk-speed in km/h
-    VELOCITY = 3.75
-
-    generate_automated_walk_table(DEPARTURE_TIME, GPX_FILE_PATH, VELOCITY)
+    # Initialize parser
+    msg = "Supported command-line args: "
+    parser = argparse.ArgumentParser(description=msg)
+
+    # Adding arguments
+    parser.add_argument('--gpx-file-name', type=str,
+                        help='Name and path to the GPX file, if not specified ./GPX/Default_Route.gpx will be used as '
+                             'default value.', default='./GPX/Default_Route.gpx')
+    parser.add_argument('--velocity', type=float, default=3.75,
+                        help='Float. Speed in km/h on which the calculation is based, default 3.75 km/h.')
+    parser.add_argument('--map-scaling', type=int,
+                        help='Integer. Scaling of the created map (e.g. 10000 for scaling of 1:10\'000), if not '
+                             'specified the scaling will be automatically chosen.')
+    parser.add_argument('--open_figure', default=False, action='store_true',
```

Comment:
--open-figure, all others have also no underscore in it.

Reply:
I've fixed it!